src/collatz.js

```diff
@@ -0,0 +1,22 @@
+import dataTypeCheck from '../src/dataTypeCheck'
+
+export default function collatz(number) {
+
+  if(!dataTypeCheck(number,'number')) {
+    return "Wrong Data Type"
+  }
+
+  let collatzArray = []
```

Comment:
Great job so far. Look at new ways of refactoring, even if it's a google search or two.